R/aggregate_results.R

```diff
@@ -76,12 +76,64 @@ aggregate_results <- function(results_list, sensitivity_analysis_vars, iter_var)
   company_pd_changes_overall <- company_pd_changes_overall %>%
     aggregate_pd_change_to_sector_level(horizon = "overall", iter_var = iter_var)
 
+  # Aggregate Crispy Results -----
+
+  crispy_output <- results_list$company_technology_npv %>%
+    dplyr::inner_join(
+      company_pd_changes_overall,
+      by = c(
+        "investor_name", "portfolio_name", "scenario_name", "scenario_geography",
+        "company_name", "ald_sector", "asset_type_arg", "baseline_scenario_arg",
+        "shock_scenario_arg", "lgd_arg", "risk_free_rate_arg", "discount_rate_arg",
+        "growth_rate_arg", "div_netprofit_prop_coef_arg", "shock_year_arg",
+        "fallback_term_arg", "use_company_terms_arg"
+      )
+    )
+
+  crispy_output <- crispy_output %>%
+    dplyr::mutate(
+      roll_up_type = dplyr::if_else(
+        .data$asset_type_arg == "bonds",
```

Comment:
I would argue that a longer comment here would be very helpful. 
Also this would created duplicates in the db if we ran caluclations for equity and loans, since there is no more way left to discriminate between them ATM I am afraid

Reply:
we can keep the asset type additionally for now and remove it, once the input is generated without the pacta step?

Reply:
sounds goodl. Maybe let us call it calculation_type actually..

Reply:
ok